docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -277,219 +284,174 @@ A closed cohort of 100,000 male and female total simulants from age 15 to 95 wil
+
+.. image:: age_specific_prev_ratio_DCIS.svg
+
+:download:`Age-specific prevalence ratio of DCIS CSV file <ratio.csv>`
+
+**LCIS**
+
+   - Age-specific prevalence ratio of DCIS = :math:`\frac{\text{age-specific prevalence of LCIS}}{\text{age-specific prevalence of breast cancer}}`
+
+   - Age-specific prevalence of DCIS 
+
+      = age-specific prevalence of ratio of DCIS X age-specific prevalence of breast cancer (prev_c429)
+
+.. image:: age_specific_prev_ratio_LCIS.svg
+
+:download:`Age-specific prevalence ratio of LCIS CSV file <ratio.csv>`
+
+
++------------------------------------------------------------------------------------------------------------------------------------------------+
+| Transition data DCIS                                                                                                                           |
++============+===============+===============+==============================================+====================================================+ 
+| Transition | Source state  | Sink state    | Value                                        | Notes                                              |
++------------+---------------+---------------+----------------------------------------------+----------------------------------------------------+
+| i_DCIS     | S             |  DCIS         | | incidence_c429                             | incidence_c429 (breast cancer) comes from como for | 
+|            |               |               | | x                                          | 2017 and forecasted for 2020-2040                  |
+|            |               |               | | incidence ratio of DCIS                    | incidence ratio of DCIS comes from MarketScan 2017 |
++------------+---------------+---------------+----------------------------------------------+----------------------------------------------------+
+| i_LCIS     | S             |  LCIS         | | incidence_c429                             | incidence_c429 (breast cancer) comes from como for | 
+|            |               |               | | x                                          | 2017 and forecasted for 2020-2040                  |
+|            |               |               | | incidence ratio of LCIS                    | incidence ratio of DCIS comes from MarketScan 2017 |
++------------+---------------+---------------+----------------------------------------------+----------------------------------------------------+
+| i_BC|DCIS  | DCIS          | BC            | :math:`\frac{\text{i_c429}}{\text{i_DCIS}}`  | i_BC= i_DCIS x i_BC|DCIS                           |
++------------+---------------+---------------+----------------------------------------------+----------------------------------------------------+
+| i_BC|LCIS  | LCIS          | BC            | :math:`\frac{\text{i_c429}}{\text{i_LCIS}}`  | i_BC= i_LCIS x i_BC|LCIS                           |
```

Comment:
Again the transition rate from LCIS to breast cancer = i_c429 / prev_LCIS, which is the incidence of breast cancer among those diagnosed with LCIS.